torch/onnx/symbolic_helper.py

```diff
@@ -173,7 +173,14 @@ def decorator(fn):
         @wraps(fn)
         def wrapper(g, *args, **kwargs):
             # some args may be optional, so the length may be smaller
-            assert len(arg_descriptors) >= len(args)
+            FILE_BUG_MSG = "If you believe this is not due to custom symbolic implementation within your code or"\
+                "an external library, please file an issue at "\
+                "https://github.com/pytorch/pytorch/issues/new?template=bug-report.md to report this bug."
+            assert len(arg_descriptors) >= len(args),\
+                f"A mismatch between the number of arguments ({len(args)}) and"\
+                f" their descriptors ({len(arg_descriptors)}) was found at symbolic function '{fn.__name__}'. "\
```

Comment:
nit: would it be possible to print the file where `fn` is at? This could be helpful figuring out which particular opset had this issue.

Reply:
I agree and I have tried before, I couldn't figure out a way that doesn't envolve the backtarce, which is printed anyway as the assert is triggered. I have tried using `inspect` and `traceback`